views/css/oobe.scss

```diff
@@ -0,0 +1,167 @@
+@import "web-os.css";
+@import "boot.css";
+@import "fonts.css";
+@import "mixins";
+@import "var";
+
+.base-pad {
+  padding-left: 4.5cm;
+}
+
+.web-os-back, html, body{
+  /*For after-install:
+  background-image: url("img/nav-back-two.png");
+  */
+  /*background-image: url("img/boot.jpg");*/
+  @include grad;
+}
+
+.web-os-oobe-back{
+  background-image: url("img/nav-back-two.png");
+}
+
+body {
+    color: $text;
+}
+
+.sub-title {
+   /* TODO: Create Subtitle class for subtitles  */
+   padding-left: 4.5cm;
+   font-size: 25px;
+}
+
+.boot-title-oobe {
+  padding-top: 1cm;
+  font-size: 60px;
+  text-align: center;
+}
+
+.oobe-next {
+  text-align: center;
+  padding-left: 71.3%;
+}
+
+form {
+  font-family: Source-Sans-Pro-Light;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.fork {
+  color: blue;
+}
+
+.system-info-build {
+  /*padding-left: 100%*/
+  text-align: right;
+  padding-top: 10%;
+  padding-right: 0.5cm;
+}
+
+.option {
+  width: 20cm;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  /*padding-right: 0.3cm;*/
+}
+
+.center {
+  text-align: center;
+}
+
+.form{
+  margin-left: 1cm;
+  margin-right: 1cm;
+}
+
+.tick-pad {
+  text-align: center;
+  margin-left: 0%;
+}
+
+.b-o-t-rec {
+  padding-right: 10cm;
+}
+
+.selection-box-box {
+  width: 30cm;
+  border-radius: 5px;
+  padding: 0.5cm;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.selection-box {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.selection-box-box:hover {
+  background-image: url('img/accept-decline.jpg');
+  width: 30cm;
+}
+
+.selection-sub {
+  padding-left: 1cm;
+}
+
+.oobe-dep-options {
+  /*font-size: 30px;*/
+}
+
+/*.par-l {
+  padding-left: 13cm;
+}*/
+
+.license {
+  font-family: Source-Sans-Pro-Light;
+  background-color: white;
+/*  width: 20cm;*/
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.lt {
+  color: black;
+  text-align: center;
+  font-family: Source-Sans-Pro-Regular;
+}
+
+.lth {
+  color: black;
+  text-align: center;
+  font-family: Source-Sans-Pro-Bold;
+}
+
+.accept {
+  border-width:2px;
+  border-style:solid;
+  border-color:white;
+  background-image: url('img/nav-back-two.png');
+  font-size: 20px;
+  border-radius: 3px;
+  margin-left: auto;
+  width: auto;
+  width: auto;
+}
+
+.accept:hover {
+  border-width:2px;
```

Comment:
Properties should be ordered background-image, border-color, border-radius, border-style, border-width, font-size, margin-left, width, width<br>Colon after property should be followed by one space